cli/command/image/build.go

```diff
@@ -643,3 +650,49 @@ func imageBuildOptions(dockerCli command.Cli, options buildOptions) types.ImageB
 		Platform:       options.platform,
 	}
 }
+
+func parseOutputs(inp []string) ([]types.ImageBuildOutput, error) {
+	var outs []types.ImageBuildOutput
+	if len(inp) == 0 {
+		return nil, nil
+	}
+	for _, s := range inp {
+		csvReader := csv.NewReader(strings.NewReader(s))
+		fields, err := csvReader.Read()
+		if err != nil {
+			return nil, err
+		}
+		if len(fields) == 1 && fields[0] == s {
+			outs = append(outs, types.ImageBuildOutput{
+				Type: "local",
+				Attrs: map[string]string{
+					"dest": s,
+				},
+			})
+			continue
+		}
+
+		out := types.ImageBuildOutput{
+			Attrs: map[string]string{},
+		}
+		for _, field := range fields {
+			parts := strings.SplitN(field, "=", 2)
+			if len(parts) != 2 {
+				return nil, errors.Errorf("invalid value %s", field)
+			}
+			key := strings.ToLower(parts[0])
```

Comment:
@thaJeztah doesn't like ToLower. He wants it strict.

Reply:
too bad for @thaJeztah then https://github.com/docker/cli/blob/774d78fcb84053f7648fae2358d7657feeb6b03b/cli/command/image/build_buildkit.go#L397

Reply:
Yes, ToNIs, noT reAlLy a FaN of toLOWER heRE

Better keep it strict, and we can loosen up if we really would see a need

Reply:
What I referred above is that everything else already is case insensitive. This includes csv for build secrets, swarm secrets, swarm config, mounts, network and ports. CSV in Dockerfile flags in daemon side is also case insensitive.

Btw, I'm not in favor of documenting it.